### PR TITLE
Add Booking Up For Beauty exercise

### PR DESCRIPTION
**Pull request by hemartin-at-redhat**

### CODE REVIEW

File: hello-world/hemartin/booking_up_for_beauty.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	}
+
+	return false
+
+}
+
+// Description returns a formatted string of the appointment time
+func Description(date string) string {
+	appointmentDate := Schedule(date)
+	return appointmentDate.Format("You have an appointment on Monday, January 2, 2006, at 15:04.")
```
Comment (davidmogar) [inline hello-world/hemartin/booking_up_for_beauty.go:45]: I didn't think about using only `time.Format` (and not `fmt.Sprintf`) for doing this. Good one.

File: hello-world/hemartin/booking_up_for_beauty.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	appointmentDate, err := time.Parse("Monday, January 2, 2006 15:04:05", date)
+	if err != nil {
+		panic(err)
+	}
+
+	if 12 <= appointmentDate.Hour() && appointmentDate.Hour() < 18 {
+		return true
+	}
+
+	return false
```
Comment (davidmogar) [inline hello-world/hemartin/booking_up_for_beauty.go:38]: Better like:
```
return 12 <= appointmentDate.Hour() && appointmentDate.Hour() < 18
```

**Review by davidmogar — COMMENTED** [review]
I would add a function to parse dates. It would make the code cleaner.

**Comment by hemartin-at-redhat** [comment]
Yeah, it would centralize the error processing too which will help maintainability.

**Pull request merged**